src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -1821,7 +1769,7 @@ public void testExceptionOnExecuteIgnoredModuleWithBadPropertyValue(
     @Test
     public void testNoProblemOnExecuteIgnoredModuleWithBadPropertyValue(
             @SysErr Capturable systemErr) throws IOException {
-        Main.main("-c", getPath("InputMainConfig-TypeName-bad-value.xml"),
+        invokeMain(0, "-c", getPath("InputMainConfig-TypeName-bad-value.xml"),
                     "", getPath("InputMain.java"));
```

Comment:
assertMainReturnCode

Reply:
Created a dedicated PR for changes in Main and MainTest:
https://github.com/checkstyle/checkstyle/pull/11173